lib/utilities/aclUtils.js

```diff
@@ -188,7 +188,7 @@ aclUtils.parseGrant = function parseGrant(grantHeader, grantType) {
 };
 
 aclUtils.isValidCanonicalId = function isValidCanonicalId(canonicalID) {
-    return /^[A-Za-z0-9]{64}$/.test(canonicalID);
+    return /^(?=.*?[a-f])(?=.*?[0-9])[a-f0-9]{64}$/.test(canonicalID);
```

Comment:
0-9 could just be \d ( for digit ) although I have seen some prefer 0-9.
I kinda wish there was \h for hex but doesn't seem like it.

```suggestion
    return /^(?=.*?[a-f])(?=.*?\d)[a-f\d]{64}$/.test(canonicalID);
```

Reply:
yeah, i found a couple of options for hex characters, but they didn't do exactly what i needed. good to know about the `/d`, but i'm gonna keep the `[0-9]` cos i think it's a clearer at a quick glance

Reply:
yea \d isn't super clear on its own, it goes well if you use all the shorthands like \w and \s.